python/lsst/sims/maf/metrics/vectorMetrics.py

```diff
@@ -0,0 +1,109 @@
+import numpy as np
+from .baseMetric import BaseMetric
+from scipy import stats
+
+__all__ = ['HistogramMetric','AccumulateMetric', 'AccumulateCountMetric',
+           'HistogramM5Metric', 'AccumulateM5Metric']
+
+
+class HistogramMetric(BaseMetric):
+    """
+    A wrapper to stats.binned_statistic
+    """
+    def __init__(self, col='night', units='Count', statistic='count',
+                 metricDtype=float, **kwargs):
+        self.statistic = statistic
+        self.col=col
+        super(HistogramMetric,self).__init__(col=col, units=units,
+                                              metricDtype=metricDtype,**kwargs)
+
+    def run(self, dataSlice, slicePoint=None):
+        dataSlice.sort(order=slicePoint['binCol'])
+        result, binEdges,binNumber = stats.binned_statistic(dataSlice[slicePoint['binCol']],
+                                                            dataSlice[self.col],
+                                                            bins=slicePoint['bins'],
+                                                            statistic=self.statistic)
+        # Make the result the same length as bins
+        result = np.append(result,0)
```

Comment:
I actually don't think this is a bad idea, but maybe a different option is to do something like what we do for oneDslicers? say that the final bin is the RHS of the last bin, instead of the LHS - so that the values returned by the histogram should indeed be one less than the length of the bins? Or does that not work with the accumulate? (or could you make the same convention for the 'accumulate' metrics -- the last 'bin' is the RHS instead of the LHS of the last bin?)